src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/LogAnalyticTests.ps1

```diff
@@ -0,0 +1,91 @@
+    $to = Get-Date -Year 2018 -Month 2 -Day 28 -Hour 9;
+    $sasuri = 'https://fakestore.blob.core.windows.net/mylogs/fakesas';
+    Assert-ThrowsContains { `
+        $result = Export-AzureRmLogAnalyticThrottledRequests -Location $loc -FromTime $from -ToTime $to -BlobContainerSasUri $sasuri -GroupByThrottlePolicy -GroupByResourceName;} `
+        "the given SAS URI";
+}
+
+<#
+.SYNOPSIS
+Test Export Log Analytic Request Rate By Interval
+#>
+function Test-ExportLogAnalyticRequestRateByInterval
+{
+    $loc = "West Central US";
+    $from = Get-Date -Year 2018 -Month 2 -Day 27 -Hour 9;
+    $to = Get-Date -Year 2018 -Month 2 -Day 28 -Hour 9;
+    $sasuri = 'https://fakestore.blob.core.windows.net/mylogs/fakesas';
+    $interval = "FiveMins";
+    Assert-ThrowsContains { `
+        $result = Export-AzureRmLogAnalyticRequestRateByInterval -Location $loc -FromTime $from -ToTime $to -BlobContainerSasUri $sasuri -IntervalLength $interval -GroupByThrottlePolicy -GroupByOperationName;} `
+        "the given SAS URI";
+}
+
+<#
+.SYNOPSIS
+Test Export Log Analytics positive scenario
+#>
+function Test-ExportLogAnalytics
+{
+    $rgname = "hyleelog";
+    $loc = "West Central US";   
+    $stoname = 'sto' + $rgname;
+    $stotype = 'Standard_GRS';
+    $container = "test";
+    $sastoken = '?fakesas'
+
+    #
+    # In order to record this test, run the following manually in a separate Azure Powershell and replace the value of SAT token.
+    #
+    #New-AzureRmResourceGroup -Name $rgname -Location $loc -Force;
+    #New-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
+    #$stoaccount = Get-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname;
+    #Set-AzureRmcurrentStorageAccount -ResourceGroupName $rgname -Name $stoname;
+    #New-AzureStorageContainer -Name $container;
+    #$sastoken = Get-AzureStorageContainer -Name $container | New-AzureStorageContainerSASToken -Permission rwdl;
+
+    try
+    {
+        $from = Get-Date -Year 2018 -Month 2 -Day 27 -Hour 9;
+        $to = Get-Date -Year 2018 -Month 2 -Day 28 -Hour 9;
+        $sasuri = "https://$stoname.blob.core.windows.net/$container$sastoken";
+        $interval = "FiveMins";
+        $result = Export-AzureRmLogAnalyticRequestRateByInterval -Location $loc -FromTime $from -ToTime $to -BlobContainerSasUri $sasuri -IntervalLength $interval -GroupByThrottlePolicy -GroupByOperationName -GroupByResourceName;
+        Assert-AreEqual "Succeeded" $result.Status;
+        $output = $result | Out-String;
+        Assert-True { $output.Contains(".csv"); }
+        #Assert-AreEqual "a" $output;
+
+        $result = Export-AzureRmLogAnalyticThrottledRequests -Location $loc -FromTime $from -ToTime $to -BlobContainerSasUri $sasuri -GroupByThrottlePolicy -GroupByOperationName -GroupByResourceName;
+        Assert-AreEqual "Succeeded" $result.Status;
+        $output = $result | Out-String;
+        Assert-True { $output.Contains(".csv"); }
```

Comment:
Is there a way to check this file to ensure it has the right format?

Reply:
I don't think checking the content of the file is a scope of Powershell cmdlets test.  It should be checked during the API test.

Reply:
Could we check the blob to ensure the file is there?

Reply:
Done.